jenkins/agent/Jenkinsfile

```diff
@@ -1,6 +1,6 @@
 def PIPELINE_NAME = "agent"
 
-def LIB_NAME= "ised-business-banking-lib"
+def LIB_NAME= "covid19-es-lib"
```

Comment:
Seeing this in a few places.  Was this intentional?  It's a little confusing given the repo is `https://github.com/bcgov/ised-business-banking-initiative.git`